modules/test/playwright/tests/object-web/main/rootModels.spec.ts

```diff
@@ -670,6 +671,160 @@ test.describe('Manage root models elements through Objects Admin', () => {
 		}
 	});
 
+	test('cannot select inheritance relationships fields in object view', async ({
+		apiHelpers,
+		editObjectViewPage,
+		objectViewPage,
+		page,
+	}) => {
+		const objectRelationships: ObjectRelationship[] = [];
+
+		try {
+			const parentObjectDefinition =
+				await apiHelpers.objectAdmin.postRandomObjectDefinition({
+					status: {code: 0},
+				});
+
+			apiHelpers.data.push({
+				id: parentObjectDefinition.id,
+				type: 'objectDefinition',
+			});
+
+			const chieldObjectDefinition =
```

Comment:
i think here you meant `childObjectDefinition` instead of `chieldObjectDefinition`